### PR TITLE
Update definitions.py

### PR DESCRIPTION
**Pull request by Charlotte-Morgan**
In progress updates for #1975

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
grateful if someone @timlinux can spot the problems for this PR for me

**Comment by timlinux** [comment]
I'm taking a look!

File: safe/definitions.py [hunk 1]
```diff
@@ -292,14 +306,21 @@
...
+        'the surface of the earth that might be exposed to a particular '
+        ' hazard. This might include crops, forest and urban areas. ')
 }
 
 exposure_people_in_building = {
     'key': 'people_in_building',
     'name': tr('people_in_building'),
     'description': tr(
-        'TBA')
+        'The <b>people in buildings/<b> expsure data is an experimental
```
Comment (timlinux) [inline safe/definitions.py]: You are missing an ' at the end of this line

**Comment by timlinux** [comment]
See also #1530

**Pull request merged**